firebase.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
```

Comment:
I know absolutely nothing about nuget/nuspec so might be nice to get a review from someone who does.